### DISCUSSION TITLE
It failed to recognized rtl sdr v4 when it is behind the usbc hub.

### DISCUSSION THREAD

**Discussion by amalgame21** [AlexandreRouma/SDRPlusPlus#1541] [Q&A]
When I plug rtl sdr v4 in with otg dongle, it display "Allow SDR++ to access Blog V4?" at app launch and it work fine.
When I plug rtl sdr v4 in with a usbc hub, it display "Allow SDR++ to access USB3.0 Card Reader?" at app launch and the sdr cannot be recognize by the app.
I am using android 15

**Comment by FlashlightET** [comment]
this happens to me as well on a pixel 6a with android 13, and I think the problem is simply the hub not having enough power/bandwidth to supply/handle the rtlsdr in a hub (which sucks because this means (thanks to Crapple back in 2016) modern phone users have to use the device speakers)

 this is simply a limitation of USB hubs and I don't think it can be fixed (I also experience this on PC)
